docs/DATA_DICTIONARY.md

```diff
@@ -42,6 +42,7 @@ Field | Description | Exportable? | Reported by | Input Type
 **Initial Call Date** | The date a patient's file was first created in DARIA. | Exportable | Autogenerated | Autogenerated
 **Urgent Flag** | This boolean flag is used to indicate an urgent case. This is used primarily for communication between case managers and has no impact on funding. | Exportable | CM Reported | Checkbox
 **When was pledge generated** | Date autogenerated from the DARIA system when a pledge is generated. | Exportable | Autogenerated | Autogenerated
+**When was pledge sent** | Date autogenerated from the DARIA system when a pledge is sent. | Exportable | Autogenerated | Autogenerated
```

Comment:
nailed it on the meaning of `pledge_sent_at` here!

let's add `fund pledged at` too, and just the titles so they match the strings in exportable:

```suggestion
**Pledge sent at** | Date autogenerated from the DARIA system when a pledge is sent. | Exportable | Autogenerated | Autogenerated
**Fund pledged at** | Date autogenerated from the DARIA system when a pledge amount changes. | Exportable | Autogenerated | Autogenerated
```